templates/dashboardTemplate.Rmd

```diff
@@ -382,16 +382,10 @@ Please apprise your patients of this schedule so that they are aware of the time
 If you see samples that should not be retracted, please upload a fix to your data or let Tom Yu from Sage know immediately.
 
 
-For patient retractions submitted between these months, a new public release will be generated that removes the affected records. The new release will be issued no later than April or October, respectively:
-
-* January-March
-* July-September
-
-For patient retractions submitted between these months, the records will be removed from the next regularly-scheduled public release in July or January, respectively:
-
-* April-June
-* October-December
+For patient retractions submitted between these months, a new public release will be generated that removes the affected records. The new release will be issued no later than July or January, respectively:
 
+* January-June
+* July-December
```

Comment:
```suggestion
* July-December

```

Reply:
Yes, I figured earlier today but it gives me enlarged July-December anyway when generating dashboard in docker container. I might have done something wrong. Thanks for pointing it out.

Reply:
Fixed.